docs/README.md

```diff
@@ -0,0 +1,19 @@
+# Project Title
```

Comment:
Nope.

The `README.md` you created is fine, but it should be outside the `docs` folder.

The `docs` folder is for documenting your tasks. For example, you can create a file called `task_1.md` in the `docs` folder and move the content in **"the process"** section of the README into it.

Inside the `task_1.md` file, you can then add other sections like the dependencies you installed for the task, the timeline (dated), the difficulty level, and so on.